sdk/digitaltwins/azure-digitaltwins-core/src/main/java/com/azure/digitaltwins/core/DigitalTwinsAsyncClient.java

```diff
@@ -90,6 +92,8 @@ public HttpPipeline getHttpPipeline() {
         return this.protocolLayer.getHttpPipeline();
     }
 
+    //region Digital Twin APIs
```

Comment:
q - should "twin" be in lower case? We've referenced it as "digital twin" everywhere.

Reply:
Sure, why not?